packages/dm-core/src/components/Table/TableRow/TableRow.tsx

```diff
@@ -47,12 +48,19 @@ export function TableRow(props: TableRowProps) {
   }
 
   function openItemAsTab() {
+    if (!props.onOpen) {
+      toast.error(
+        'Invalid UiRecipes. The table plugin was not passed an "onOpen()"-function.'
+      )
+      return
+    }
```

Comment:
Should be allowed not to pass an "onOpen" function

Reply:
testet bare kjapt med `if(true) ...` på linje 51
<img width="1257" alt="image" src="https://github.com/equinor/dm-core-packages/assets/43639886/aa2c4a9d-0b87-4815-9e8f-ff468ac87e59">